httpcore5/src/main/java/org/apache/hc/core5/http/ssl/TLS.java

```diff
@@ -94,4 +94,14 @@ public static ProtocolVersion parse(final String s) throws ParseException {
         return enabledProtocols.toArray(new String[0]);
     }
 
+    /**
+     * Check if a given protocol is enabled.
+     *
+     * @return {@code true} if the given protocol is enabled, otherwise return {@code false}.
+     * @since 5.2
+     */
+    public static boolean isEnable(final String protocol) {
```

Comment:
@arturobernalg `#isSecure` should be a better name. Method description should be: `Check if a given protocol is considered secure and is enabled by default.`

Reply:
changed.
TY